src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -69,6 +82,58 @@ public void Dispose()
             this.Dispose(true);
         }
 
+        internal void ApplyLibraryResource(Resource libraryResource)
```

Comment:
`ApplyLibraryResource` is used only `JaegerSpanExporterTests` . As we are removing span exporter, do we require this method?

Reply:
@cijothomas , should i maintain this logic or should i remove?

Reply:
@rajkumar-rangaraj , since it was approved, it went with this function. Don't know yet what is the answer for this.